typing/typeclass.ml

```diff
@@ -580,8 +597,8 @@ let rec class_field self_loc cl_num self_type meths vars arg cf =
   Builtin_attributes.warning_scope cf.pcf_attributes
     (fun () -> class_field_aux self_loc cl_num self_type meths vars arg cf)
 
-and class_field_aux self_loc cl_num self_type meths vars
-    (val_env, met_env, par_env, fields, concr_meths, warn_vals, inher,
+and class_field_aux self_loc cl_num self_type meths vars (* TODO: tuple to record *)
+    (val_env, met_env, par_env, fields, concr_meths, warn_vals, inher, 
```

Comment:
Have a care committing TODO comments - it'll be cleaner in the final history if they're not there at all (I personally find `git add -p` more helpful than `git add -u`, but it depends on how pernickety you are with your commit-crafting fu!)

Reply:
Thank you for taking time to review my PR! Would you recommend from committing TODOs even if they are removed before the merge?

Reply:
I'd recommend against putting them in the same commits as other things, unless your intention is to squash everything together at the end.